lib/inferno/apps/web/index.html.erb

```diff
@@ -1,3 +1,5 @@
+<style></style>
```

Comment:
I had this style tag populated with classes (like `wrapper`/`banner`/`app`), but ended up not setting any style attributes for them. However, removing this empty style tag or the class assignments from the div tags causes everything to go wonky. I'm gonna figure out what's going on and update.